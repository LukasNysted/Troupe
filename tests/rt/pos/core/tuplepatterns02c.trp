let fun f () = (1,3,7) in
  case f () of
    (1,3) => false
  end