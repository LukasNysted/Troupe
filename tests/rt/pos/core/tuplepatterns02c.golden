2024-02-14T11:33:49.445Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
Runtime error in thread d300cb00-f4c2-46e3-82eb-fe692175bd42@{}%{}
>> pattern match failure in case expression at :2:3
